docs/reference/cluster/nodes-reload-secure-settings.asciidoc

```diff
@@ -22,9 +22,37 @@ The first command reloads the keystore on each node. The seconds allows
 to selectively target `nodeId1` and `nodeId2`. The node selection options are
 detailed <<cluster-nodes,here>>.
 
-Note: It is an error if secure settings are inconsistent across the cluster
-nodes, yet this consistency is not enforced whatsoever. Hence, reloading specific
-nodes is not standard. It is only justifiable when retrying failed reload operations.
+==== Reload Password Protected Secure Settings
+
+When the elasticsearch keystore is password protected and not simply obfuscated, the password for the keystore needs to be provided in the request to reload the secure settings.
+Reloading the settings for the whole cluster assumes that all nodes keystores are protected with the same password and is only allowed when
+{ref}/configuring-tls.html#tls-transport[node to node communications are encrypted]
+
+[source,js]
+--------------------------------------------------
+POST _nodes/reload_secure_settings
+{
+  "reload_secure_settings": "s3cr3t" <1>
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+<1> The password that the elasticsearch keystore is encrypted with.
+
+Alternatively the secure settings can be reloaded on a per node basis, locally accessing the API and passing the node-specific elasticsearch keystore password.
+
+[source,js]
+--------------------------------------------------
+POST _nodes/_local/reload_secure_settings
+{
+  "reload_secure_settings": "s3cr3t" <1>
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+<1> The password that the elasticsearch keystore is encrypted with.
+
+Note:It is an error if secure settings are inconsistent across the cluster nodes, yet this consistency is not enforced whatsoever.Hence, reloading specific nodes is not standard.It is only justifiable when retrying failed reload operations.
```

Comment:
Should this be a real asciidoc "Note"? Also, seem to have lost some spacing between sentences in moving down here.